src/main/java/edu/ie3/util/quantities/QuantityUtil.java

```diff
@@ -131,42 +191,89 @@ public static boolean quantityIsEmpty(Quantity<?> that) {
+   * does not matter. Throws a NullPointerException if any quantity is null, as null is not to be
+   * expected as any known empty value should be replaced by an EmptyQuantity.
+   *
+   * <p>isTheSameConsideringEmpty(1 km, 1000 m) // false <br>
+   * isTheSameConsideringEmpty(1.0 km, 1 km) //true
+   *
+   * @param a First quantity to compare
+   * @param b Second quantity to compare
+   * @param quantityTolerance permissible absolute tolerance
+   * @param <Q> Type of the Quantity
+   * @return true, if both are an EmptyQuantity or if they are equal
+   * @throws NullPointerException if any quantity is null
+   */
+  public static <Q extends Quantity<Q>> boolean equals(
+      Quantity<Q> a, Quantity<Q> b, double quantityTolerance) {
     Objects.requireNonNull(
         a, "There was an error while comparing quantities: quantity a was not set");
     Objects.requireNonNull(
         b, "There was an error while comparing quantities: quantity b was not set");
-    if (!QuantityUtil.quantityIsEmpty(a)) {
-      if (QuantityUtil.quantityIsEmpty(b)) return false;
-      return a.equals(b);
-    } else return QuantityUtil.quantityIsEmpty(b);
+    if (!QuantityUtil.isEmpty(a)) {
+      if (QuantityUtil.isEmpty(b)) return false;
+      if (!a.getUnit().equals(b.getUnit())) return false;
+      return Math.abs(a.getValue().doubleValue() - b.getValue().doubleValue()) <= quantityTolerance;
+    } else return QuantityUtil.isEmpty(b);
   }
 
   /**
    * Compares two {@link Quantity}s, if they are equivalent. Returns true only if both quantities
    * are of type {@link EmptyQuantity} or if they represent equivalent values. Throws a
-   * NullPointerException if the any quantity is null, as null is not to be expected as any known
-   * empty value should be replaced by an EmptyQuantity.
+   * NullPointerException if any quantity is null, as null is not to be expected as any known empty
+   * value should be replaced by an EmptyQuantity. Internally calls {@link
+   * QuantityUtil#isEquivalentConsideringEmpty(ComparableQuantity, ComparableQuantity, double)} with
+   * a default tolerance of {@value DEFAULT_TOLERANCE}
    *
-   * <p>isEquivalentConsideringEmpty(1 km, 1000 m)// true <br>
-   * isEquivalentConsideringEmpty(1.0 km, 1 km) // true
+   * <p>isEquivalentConsideringEmpty(1 km, 1000 m, 0)// true <br>
+   * isEquivalentConsideringEmpty(1.0 km, 1 km, 0) // true
    *
    * @param a First quantity to compare
    * @param b Second quantity to compare
    * @param <Q> Type of the Quantity
-   * @return true, if both are an EmptyQuantity or if they are equivalent
+   * @return true, if both are an EmptyQuantity or both quantities' values differ less then the
+   *     given tolerance else false (based on a's unit)
    * @throws NullPointerException if any quantity is null
    */
   public static <Q extends Quantity<Q>> boolean isEquivalentConsideringEmpty(
```

Comment:
Please move the equivalence methods upwards, in order to have equivalence and equality methods closely together.